scipy/docker/install.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set +e
```

Comment:
Did you mean `set -e`?
Also consider adding `u` and `o pipefail`

Reply:
Done!